### ISSUE TITLE
ref-transcript-mismatch-reporter ZeroDivisionError: float division by zero

### ISSUE THREAD

**Issue by weilinwu97** [griffithlab/VAtools#63]
Hi I am using griffithlab/vatools:latest docker image to annotate my vcf. However, I ran into this error:

```
2023-07-27T02:16:47.901908883Z Traceback (most recent call last):
2023-07-27T02:16:47.901930024Z   File “/usr/local/bin/ref-transcript-mismatch-reporter”, line 11, in <module>
2023-07-27T02:16:47.901936846Z     sys.exit(main())
2023-07-27T02:16:47.901939332Z   File “/usr/local/lib/python3.6/dist-packages/vatools/ref_transcript_mismatch_reporter.py”, line 215, in main
2023-07-27T02:16:47.901941899Z     “Percentage of processable transcripts with mismatched annotations: {}%\n”.format(round(float(filtered_transcript_count)/float(processable_transcript_count)*100, 2)) +
2023-07-27T02:16:47.901944955Z ZeroDivisionError: float division by zero
```

This is my command

```Shell
ref-transcript-mismatch-reporter -f hard -o <output_file_name> <input.vcf.gz>
```

where the input for testing is [input.vcf.gz](https://github.com/griffithlab/pVACtools/files/12654838/input.vcf.gz)

Seems like when there is no processable transcripts (`processable_transcript_count = 0`) it will throw this error. 

Cheers, 
Weilin

**Comment by susannasiebert** [comment]
Unfortunately, the attached VCF is not VEP-annotated. Can you please share the VEP-annotated version of this VCF?

**Comment by weilinwu97** [comment]
Sorry, here is the VEP version. 
[input.vep.vcf.gz](https://github.com/griffithlab/VAtools/files/12678895/input.vep.vcf.gz)

thanks, 
Weilin

**Comment by susannasiebert** [comment]
This error has been resolved in version 5.1.0. Please let me know if you're still running into problems.
